mobile/lib/services/api.service.dart

```diff
@@ -35,6 +35,9 @@ class ApiService implements Authentication {
   late MemoriesApi memoriesApi;
 
   ApiService() {
+    // The below line ensures that the api clients are initialized when the service is instantiated
+    // This is required to avoid late initialization errors when the clients are access before the endpoint is resolved
+    setEndpoint('');
```

Comment:
Did we run into this issue previously? What is the new implementation that triggered this condition?